circle.yml

```diff
@@ -20,10 +20,16 @@ test:
   pre:
     - chmod -R 777 $CIRCLE_TEST_REPORTS
   override:
-    # Run Lint checks
+    # Run Python lint checks
     - >
-      docker run -v $CIRCLE_TEST_REPORTS:/test_artifacts
+      docker run --net host
+      -v $CIRCLE_TEST_REPORTS:/test_artifacts
       normandy:build flake8 --output-file /test_artifacts/flake8.txt normandy
+    # Run JS lint checks
+    - >
+      docker run --net host
+      -v $CIRCLE_TEST_REPORTS:/test_artifacts
+      normandy:build node_modules/.bin/eslint normandy -f junit -o /test_artifacts/eslint.xml
```

Comment:
It'd be neat if there was an `npm run lint` that we could use here in case we ever want to change how we lint without having to modify all the different places we run linting.